app/styles/play/level.sass

```diff
@@ -259,6 +259,10 @@ $level-resize-transition-time: 0.5s
     right: 15px
     font-size: 30px
 
+  .ad-container
+    width: 100%
+    height: 90px
+    text-align: center
```

Comment:
Yea, that's not great for a significant chunk of users.  It looks fine if you zoom out a bit for this specific scenario, so I'll go with that for now.